com.calibre_ebook.calibre.yaml

```diff
@@ -10,6 +10,7 @@ finish-args:
   - --share=network
   - --socket=x11
   - --talk-name=org.freedesktop.Notifications
+  - --devices=all
```

Comment:
The flag is `--device=all`, so I guess you never even tested this.

Reply:
Please keep the list of `finish-args` sorted alphabetically. Thanks.

Reply:
Done, will keep in mind in the future.